content/en/docs/tasks/run-application/configure-pdb.md

```diff
@@ -111,6 +114,12 @@ of the evicted pod. `minAvailable` can be either an absolute number or a percent
 of the number of pods from that set that can be unavailable after the eviction. 
 It can be either an absolute number or a percentage.
 
+{{< note >}}
+The behavior for an empty selector differs between the policy/v1beta1 and policy/v1 APIs for
+PodDisruptionBudgets. For policy/v1beta1 an empty selector matches zero pods, while
+for policy/v1 an empty selector matches every pod in the namespace.
+{{< /note >}}
+
 {{< note >}}
```

Comment:
Consider omitting the “For versions 1.8 and earlier” note as v1.8 is long out of support.

Reply:
Removed the section.